contrib/utilities/indent_common.sh

```diff
@@ -259,7 +259,7 @@ fix_permissions()
 export -f fix_permissions
 
 #
-# Collect all files found in a list of directories "${1}$" matching a
+# Collect all files found in a list of directories "${1}" matching a
 # regular expression "${2}$", and process them with a command "${3}" on 10
```

Comment:
```suggestion
# regular expression "${2}", and process them with a command "${3}" on 10
```

Reply:
No, that's actually correct. The regular expression we use really is `${2}$`.

Reply:
Nevermind, this is line 263, not line 320 - this one should be changed!